internal/shim-sgx/Cargo.toml

```diff
@@ -29,7 +29,7 @@ noted = "^1.0.0"
 xsave = { version = "2.0.2" }
 rcrt1 = "1.0.0"
 lset = "0.2"
-sgx = "0.3"
+sgx = { path = "/home/jarkko/Projects/Enarx/sgx" }
```

Comment:
Please upload your changes to github and point to that commit here, so we can reproduce it.

Reply:
The pull request is here: https://github.com/enarx/sgx/pull/62.

So I guess it is like `sgx = { git = "https://github.com/jarkkojs/sgx/tree/feat/sinfo/eaccept" }`?

Reply:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories